library/NotificationCenter/Model/Message.php

```diff
@@ -62,6 +62,17 @@ public function send(array $arrTokens, $strLanguage = '', array $arrAttachments
             }
         }
 
+        $cpLanguage = $cpLanguage === '' ? $GLOBALS['TL_LANGUAGE'] : $cpLanguage;
+        if (($objLanguage = Language::findByMessageAndLanguageOrFallback($this, $strLanguage)) === null) {
+            \Contao\System::log(sprintf('Could not find matching language or fallback for message ID "%s" and language "%s".', $this->id, $cpLanguage), __METHOD__, TL_ERROR);
+
+            return false;
+        }
+
+        // Switch to the language of the notification
+        $currentLanguage = $GLOBALS['TL_LANGUAGE'];
```

Comment:
Sould we save the current language of the symfony translator, too?
Or is it always the same as `TL_LANGUAGE`?